package.json

```diff
@@ -1,13 +1,19 @@
 {
-    "name": "prestashop-common",
-    "version": "1.0.0",
-    "scripts": {
-        "install-test": "npm -C tests/js install",
-        "upgrade-test": "npm -C tests/js update",
-        "test": "cd tests/js && grunt test",
-        "test-short": "cd tests/js && grunt test-short",
-        "test-verbose": "cd tests/js && grunt --stack test-verbose"
-    },
-    "author": "PrestaShop",
-    "license": "OSL-3.0"
+  "name": "prestashop-common",
+  "version": "1.0.0",
+  "scripts": {
+    "lint": "eslint -c .eslintrc.js --ext .js,.vue ."
+  },
+  "author": "PrestaShop",
+  "license": "OSL-3.0",
+  "devDependencies": {
+    "babel-eslint": "^8.2.6",
+    "eslint": "^5.2.0",
+    "eslint-config-airbnb": "^17.0.0",
+    "eslint-config-airbnb-base": "^13.0.0",
+    "eslint-plugin-html": "^4.0.5",
+    "eslint-plugin-import": "^2.13.0",
+    "eslint-plugin-jsx-a11y": "^6.1.1",
+    "eslint-plugin-react": "^7.10.0"
```

Comment:
Why this? we dont' use react nor jsx. It would make more sense to add the vue plugin.

Reply:
These are airbnb dependencies:
```
eslint:config-file Loading /home/got/projects/prestashop/source/develop/node_modules/eslint-config-airbnb/rules/react-a11y.js +4ms
  eslint:config-file Loading JS config file: /home/got/projects/prestashop/source/develop/node_modules/eslint-config-airbnb/rules/react-a11y.js +0ms
```

Reply:
> If you don't need React, see eslint-config-airbnb-base.

[Source](https://www.npmjs.com/package/eslint-config-airbnb)